shared/Rakefile

```diff
@@ -70,6 +70,30 @@ namespace :test do
     t.test_files = FileList["test/showcase/**/*_test.rb"]
     t.warning = false
   end
+
+  desc "Runs tests for all expected output gems."
+  task :output do
+    {
+      gapic: [:showcase, :garbage],
+      cloud: [:language, :speech, :vision]
+    }.each do |generator, gems|
+      directory = (generator == :gapic) ? "gapic/templates" : generator
+      gems.each do |gem|
+        Dir.chdir "./output/#{directory}/#{gem}" do
+          puts ""
+          puts "#"*42
+          puts "# Testing output for #{generator}'s #{gem}"
+          puts "#"*42
+
+          Bundler.with_unbundled_env do
+            sh "bundle install"
+            puts "Running rubocop for #{gem}"
+            sh "bundle exec rake rubocop"
```

Comment:
I'd love to expand this in a follow-up PR to run `rake ci` where available. (I pulled the branch and tried it locally, the gapic gems don't have `ci` and the cloud gems have some failures.)

Reply:
Yes, my intention is to move towards that as more things from the generator output start passing. But it might end up be part of the generator output tests in the instead of here in the shared Rakefile.